TypeInjections/Tool/Tool.fs

```diff
@@ -35,19 +38,17 @@ module Tool =
           [<Option('c', "clone", Required = false, HelpText = "input a git URL")>]
           url: string
           [<Option('f', "force", Required = false, HelpText = "overwrite the Old and New folders in the Output folder")>]
-          force: bool }
-
-    let checkInputs (paths: string list) =
-        for a in paths do
-            // ensure folders are in computer
-            if Directory.Exists(a) = false then
-                failwith $"{a} is not found"
-            // make sure folder has at least 1 dafny file
-            // (typeCart is already smart enough to ignore non-dafy files)
-            let dafFiles = Directory.GetFiles(a, "*.dfy")
-
-            if dafFiles.Length = 0 then
-                failwith $"{a} does not contain any dafny files"
+          force: bool
+          [<Option('i', "ignore", Required = false, HelpText = "specify filename(s) you want to ignore")>]
+          ignore: seq<string>
+          }
+
+    type parsedOption(oldFolder: string, newFolder: string, outFolder: string, ignore: string list) =
```

Comment:
Type names should be uppercase. Why not use a record type that exposes its members automatically?
```fsharp
type ParsedOption = { oldFolder: string; newFolder: string; outFolder: string; ignore: string list }
```

Reply:
Fixed